spark/src/main/scala/geotrellis/spark/tiling/ZoomedLayoutScheme.scala

```diff
@@ -116,13 +116,15 @@ class ZoomedLayoutScheme(val crs: CRS, val tileSize: Int, resolutionThreshold: D
 
   def zoomOut(level: LayoutLevel) = {
     val layout = level.layout
+    val newZoom = level.zoom - 1
+    val newSize = math.pow(2, newZoom).toInt
```

Comment:
Not sure we need to keep the /2 *2 functionality, that was just poor implementation I think. The way this scheme is intended to be used is with square, power of two, layouts. The fact that it may work under some conditions for non-power of two is incidental and would not have produce a usably correct result if it completed. We're just fixing and testing an implementation here right now.